docs/reference/glossary.md

```diff
@@ -104,10 +155,18 @@ A security model for certain layer 2 solutions where, to increase speed, transac
 
 ## Z
 
-**Zero-knowledge proof**
+### Zero-knowledge proof
+
+A zero-knowledge proof is a cryptographic method that allows an individual to prove that a statement is true without conveying any additional information. This is commonly associated with privacy, but is great for _scaling_, because it reduces the amount of information you have to provide. This is the validity proof that the “powerful computer” must provide for [verifiable computation](#v).
```

Comment:
but is great for _scaling_ **blockchain networks through rollups**, because it reduces the amount of information you have to provide **to lower layers**.